### DISCUSSION TITLE
Wacom touch pen does not work

### DISCUSSION THREAD

**Discussion by razielaka** [rustdesk/rustdesk#10097] [General]
### Bug Description

When using a Wacom One CTL-672 tablet connected to a computer via RustDesk, the touch pen functionality does not work. Specifically, the pen cannot write, click, or perform any actions, including those mapped to the pen's buttons. The issue occurs only when the tablet is used through RustDesk, and it works fine in other contexts.

### How to Reproduce

    Connect a Wacom One CTL-672 tablet to Computer A.
    Set up remote control between Computer A and Computer B using RustDesk.
    Try to use the touch pen from Computer A to interact with Computer B.

### Expected Behavior

The Wacom One CTL-672 touch pen should work as expected through RustDesk, allowing for writing, clicking, and use of the pen buttons on Computer B.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 11 on both sides

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.1

### Screenshots

Not applicable

**Comment by d3x0r** [comment]
Yes, pen inputs do not transfer to the remote system.  I'm using a different pen tablet, and the mouse moves, and hoverover events get generated on the remote no pen down or pen-button events are transferrred.
